app/src/main/java/io/gnosis/safe/notifications/NotificationManager.kt

```diff
@@ -138,18 +143,19 @@ class NotificationManager(
             }
             is PushNotification.IncomingToken -> {
                 if (pushNotification.tokenId != null) {
-                    title = context.getString(R.string.push_title_received_erc721)
+                    title = context.getString(R.string.push_title_received_erc721, safe.chain.name)
                     text = context.getString(R.string.push_text_received_erc721, safeName)
                 } else {
-                    title = context.getString(R.string.push_title_received_erc20)
+                    title = context.getString(R.string.push_title_received_erc20, safe.chain.name)
                     text = context.getString(R.string.push_text_received_erc20, safeName)
                 }
                 intent = txListIntent(safe)
             }
             is PushNotification.IncomingEther -> {
-                title = context.getString(R.string.push_title_received_eth)
-                val value = balanceFormatter.shortAmount(pushNotification.value.convertAmount(TokenRepository.NATIVE_CURRENCY_INFO.decimals))
-                text = context.getString(R.string.push_text_received_eth, safeName, value)
+                val currencySymbol = safe.chain.currency.symbol
```

Comment:
@elgatovital can you please update the original ticket / document what will be the new push notification message format so that we have the same on iOS when we implement it?

Reply:
sure